saleor/static/dashboard-next/navigation/mutations.ts

```diff
@@ -0,0 +1,143 @@
+import gql from "graphql-tag";
+import { TypedMutation } from "../mutations";
+import { menuItemFragment, menuItemNestedFragment } from "./queries";
+import {
+  MenuBulkDelete,
+  MenuBulkDeleteVariables
+} from "./types/MenuBulkDelete";
+import { MenuCreate, MenuCreateVariables } from "./types/MenuCreate";
+import { MenuDelete, MenuDeleteVariables } from "./types/MenuDelete";
+import {
+  MenuItemCreate,
+  MenuItemCreateVariables
+} from "./types/MenuItemCreate";
+import {
+  MenuItemUpdate,
+  MenuItemUpdateVariables
+} from "./types/MenuItemUpdate";
+import { MenuUpdate, MenuUpdateVariables } from "./types/MenuUpdate";
+
+const menuCreate = gql`
+  mutation MenuCreate($input: MenuCreateInput!) {
+    menuCreate(input: $input) {
+      errors {
+        field
+        message
+      }
+      menu {
+        id
+      }
+    }
+  }
+`;
+export const MenuCreateMutation = TypedMutation<
+  MenuCreate,
+  MenuCreateVariables
+>(menuCreate);
+
+const menuBulkDelete = gql`
+  mutation MenuBulkDelete($ids: [ID]!) {
+    menuBulkDelete(ids: $ids) {
+      errors {
+        field
+        message
```

Comment:
This could be a fragment.

Reply:
Could be, but it would save me... oh wait, nothing.

Reply:
If you're duplicating error query all over the dashboard it could indeed be turned into a fragment. That's just a good practice to extract common stuff.

Reply:
Ok then #4111